### ISSUE TITLE
Bad Classification Results on the Generated Embedding 

### ISSUE THREAD

**Issue by rzamarefat** [IrvingMeng/MagFace#31]
Hi, thank you for this repo and open-sourcing it. I have generated embedding of approximately 7k perfectly-matched-to-face images (such as the following) and then tried  to classify them using SVM (sklearn --> linear and RBF kernels) but I have got around 85% accuracy on the task. On the other hand when I used FACENET-generated embedding on the exact same SVM I got around 94%. Please tell me if I am doing sth wrong because the results are not at all SOTA. 
![177d6a19bd5f42558610c140ab387cf1](https://user-images.githubusercontent.com/79300456/138084027-cb423fcd-3640-4c38-8ec2-6823c419eb69.jpg)
![9981c2614b1747fc8601c0bd518ed105](https://user-images.githubusercontent.com/79300456/138084032-0def35c0-44ca-477b-9d35-77e35da22bec.jpg)

**Comment by IrvingMeng** [comment]
Again, please re-align your images before using the pre-trained model or re-train a MagFace model based on your aligned inputs.
